.github/workflows/check.yaml

```diff
@@ -28,13 +32,12 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - juju-channel: "3.1/stable"
-            command: "make functional"
+          - juju-channel: "3.4/stable"
+            command: "TEST_JUJU3=1 make functional"  # using TEST_JUJU3 due https://github.com/openstack-charmers/zaza/commit/af7eea953dd5d74d3d074fe67b5765dca3911ca6
```

Comment:
Now, I'm curious how this works, since you did not set `TEST_JUJU3` or `TEST_*` passenv in tox.

Reply:
Yeah this is weird, I printed the env and saw it is picked by tox without being set in passenv... I just added it to passenv for now.